README.md

```diff
@@ -126,6 +132,37 @@ The token attributions are now computed only for the area specified in the image
  ('background', 23.05)]
 ```
 
+> Note: if you encounter VRAM constraint, it is possible to limit or disable the token attribution as follows:
+
+
+```python
+from diffusers_interpret import StableDiffusionPipelineExplainer
+
+explainer = StableDiffusionPipelineExplainer(
+    pipe,
+    
+    # We pass `True` in here to be able to have a higher `n_last_diffusion_steps_to_consider_for_attributions` in the cell below
+    gradient_checkpointing=True 
+)
+
+prompt = "A cute corgi with the Eiffel Tower in the background"
+
+generator = torch.Generator(device).manual_seed(2023)
+with torch.autocast('cuda') if device == 'cuda' else nullcontext():
+    output = explainer(
+        prompt, 
+        num_inference_steps=50, 
+        generator=generator,
+        height=448,
+        width=448,
+        
+        # for this model, the GPU VRAM usage will raise drastically if we increase this argument. feel free to experiment with it
+        # if you are not interested in checking the token attributions, you can pass 0 instead
+        n_last_diffusion_steps_to_consider_for_attributions=5
+    )
+
+```
+

```

Comment:
I was thinking about doing something different. What do you think?
```suggestion
If you are having GPU memory problems, try reducing `n_last_diffusion_steps_to_consider_for_attributions`, `height`, `width` and/or `num_inference_steps`.
```python
output = explainer(
    prompt, 
    num_inference_steps=15,
    generator=generator,
    height=448,
    width=448,
    n_last_diffusion_steps_to_consider_for_attributions=5
)
```
You can completely deactivate token/pixel attributions computation by passing `n_last_diffusion_steps_to_consider_for_attributions=0`.  

Gradient checkpointing also reduces GPU usage, but makes computations a bit slower:
```
explainer = StableDiffusionPipelineExplainer(pipe, gradient_checkpointing=True)
```
```